mpu/mpu.go

```diff
@@ -62,3 +62,21 @@ func ReadMPU() (pitch, roll, heading float32, err error) {
 	}
 	return
 }
+
```

Comment:
we can make this a little easier on the client and return a struct, e.g.:

    type RawData struct {
        gx, gy, gz, ax, ay, az, mx, my, mz float32
    }

    func ReadMPURaw() (d RawData, err error) {
        i := int(C.read_mpu_raw(
            (*C.float)(unsafe.Pointer(&d.gx)),
            (*C.float)(unsafe.Pointer(&d.gy)),
        ...
    }